Sources/_RegexParser/Regex/Parse/LexicalAnalysis.swift

```diff
@@ -597,7 +597,7 @@ extension Source {
         }.value
 
         // In multi-line literals, the quote may not span multiple lines.
-        if context.syntax.contains(.multilineExtendedSyntax),
+        if context.syntax.contains(.multilineLiteral),
```

Comment:
Just to double check. There's a difference between a multi-line source string containing regex syntax and a multi-line literal. The former is fine, it's just that the lines in a literal are treated differently

Reply:
Indeed, and I've renamed `multilineLiteral` to `multilineCompilerLiteral`